### PR TITLE
subtask\OL-52 Added Spatie Library functionality

### PR DESCRIPTION
**Pull request by MatijaBojanic**

### CODE REVIEW

File: app/Http/Controllers/AuthorController.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        $validator = Validator::make($request->all(), [
+            'first_name' => 'required|string|max:255',
+            'last_name' => 'required|string|max:255',
+            'biography' => 'nullable|string',
+            'picture' => 'nullable|image|mimes:jpeg,png,jpg,gif|max:2048',
+        ]);
+
+        if ($validator->fails()) {
+            return response()->json(['errors' => $validator->errors()], 422);
+        }
```
Comment (MatijaBojanic) [inline app/Http/Controllers/AuthorController.php]: Unless you want specific logic to handle validation errors, you can just use
```
$request->validate([
            'first_name' => 'required|string|max:255',
            'last_name' => 'required|string|max:255',
            'biography' => 'nullable|string',
            'picture' => 'nullable|image|mimes:jpeg,png,jpg,gif|max:2048',
        ]);
```

File: database/migrations/2024_07_29_093802_create_authors_table.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('authors', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
```
Comment (MatijaBojanic) [inline database/migrations/2024_07_29_093802_create_authors_table.php:16]: Seems like you forgot to add other fields, like first_name
@VoxMne

Comment (VoxMne) [reply database/migrations/2024_07_29_093802_create_authors_table.php:16]: Ispravljeno na OL Delete Author commitu

**Comment by VoxMne** [comment]
@MatijaBojanic evo novi sam commit uradio pa vidi  moze li on da se merguje sad?

**Pull request closed without merging**